production/helm/loki-stack/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "loki.name" -}}
```

Comment:
Should be `loki-stack.name`, etc. for these to not be confused with the loki chart which has the same functions.